Braintree.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Do we want to remove this file?

Reply:
Sorry yes - lots of merge conflicts earlier

Reply:
I think that was my bad - I probably should have merged mine in last since it was also nested in this structure. 🙈

Reply:
All good, that package.resolves is always a PITA!